common/cards/default/hermits/rendog-rare.ts

```diff
@@ -58,12 +58,17 @@ const RendogRare: Hermit = {
 		if (!newAttack) return null
 
 		const attackName = mockedAttack.attackName
-		newAttack.updateLog(
-			(values) =>
-				`${values.attacker} ${values.coinFlip ? values.coinFlip + ', then ' : ''} attacked ${
-					values.target
-				} with $v${mockedAttack.hermitName}'s ${attackName}$ for ${values.damage} damage`,
-		)
+		newAttack.updateLog((values) => {
+			if (
+				values.attack.getDamageMultiplier() === 0 ||
+				values.attack.target?.getHermit() === null
```

Comment:
What is `values.attack.target?.getHermit()` is `undefined`

Reply:
According to the types this shouldn't be possible but I can change it to a !

Reply:
Don't change it to a !. Write an assertion if you are confident it can't happen.

Reply:
nah i was wrong it can be undefined lol